lib/workers/repository/update/pr/body/notes.ts

```diff
@@ -38,11 +38,5 @@ export function getPrExtraNotes(config: BranchConfig): string {
     );
   }
 
-  if (config.isPin) {
-    res += emojify(
-      `:pushpin: **Important**: Renovate will wait until you have merged this Pin PR before creating any *upgrade* PRs for the affected packages. Add the preset \`:preserveSemverRanges\` to your config if you instead don't wish to pin dependencies.\n\n`
```

Comment:
Do we want to keep the note about using `:preserveSemverRanges`?

Reply:
Good idea, yes

Reply:
@HonkingGoose  @rarkins Like this ?
![image](https://user-images.githubusercontent.com/98313426/160853836-13834f3b-bd37-472e-80b7-8a3360a42baf.png)

Reply:
@HonkingGoose @rarkins like this?
![image](https://user-images.githubusercontent.com/98313426/160855211-694ecb63-35ef-4d74-9342-cf732acdec76.png)